docker-compose.dev.yml

```diff
@@ -10,15 +10,14 @@ services:
     ports:
       - "5432:5432"
 
-  livebook:
-    image: livebook/livebook:0.2.0
-    container_name: liveman_livebook
+  liveman:
+    image: nimblehq/liveman:0.1.0
+    container_name: liveman_liveman
```

Comment:
Just minor thing but we can rename the container names:
- `liveman_db` -> `liveman_demo_api_db`
- `liveman_livebook` -> `liveman_demo_api_liveman`